### DISCUSSION TITLE
FV3 output

### DISCUSSION THREAD

**Discussion by mjhossen** [ufs-community/ufs-weather-model#1623] [Q&A]
I need to extract GeoVals for water vapor, relative humidity, ozone thickness etc. Is it possible to get values of those variables? Is there any documents where I can find variable list?

**Comment by ShawnCebulaNOAA** [comment]
Hello!
You should be able to find what you're looking for in our weather model documentation, available [here on ReadTheDocs](https://ufs-weather-model.readthedocs.io/en/latest/Introduction.html).  Hope this helps!

Shawn Cebula, EPIC End User Support

**Reply by ShawnCebulaNOAA** [reply]
Bit more information, in case it helps - I believe [this section](https://ufs-weather-model.readthedocs.io/en/latest/InputsOutputs.html#output-files) might be most useful.  What you're looking for should be in the FV3 output files.

**Comment by mjhossen** [comment]
I am wondering how to run fv3 using soca-science workflow. Any idea?
